website/src/content/docs/general/support-policy.md

```diff
@@ -1,41 +1,91 @@
+- We do not support any Cloud providers or OpenShift.
+
+## ClusterTool Support Policy
+
+- We offer support for all ClusterTool functions, except those under `adv` and `charts`
+- If you deviate from default configurations significantly, we cannot guarantee a bootstrap will work and support will be limited on a case-by-case basis.
+- We do provide support for issues regarding talconfig.yaml, as long as ClusterTool is used for the deployment.
+- We do provide *basic* assistance with [Talos OS](./guides/talos), when deployed or maintained through ClusterTool (upgrades, config changes etc)
+- We do provide support for FluxCD setup and layout, as long as ClusterTool is used for the deployment.
+
+## Kubernetes Support Policy
+
+- We do not offer support for any component of kubernets that is not made by TrueCharts.
+- This includes charts by other parties or manually created kubernetes manifests and deployments.
+- We do not offer support for any component of kubernets that is not made by TrueCharts.
+
+## Helm Chart Support Policy
+
+- We offer support for all features available in our helm-charts, including all options in the common-chart.
+- We offer [basic documentation](./guides/helm) and support for installing, upgrading and editing our Helm-Charts using Helm.
+- We do not support external secret management outside of values.yaml
+- We offer support for all our charts trains, except `incubator`, `dev`, and `test`.
+
+:::note
 
-## What's Explicitly Not Covered By Support:
+Some (common-chart) features might not work correctly with all charts.
 
-- Network configuration
-- CloudFlare proxying
-- Operating-Systems, outside of Talos deployed using ClusterTool
-- Helm 101 (e.g. "how do I install/upgrade/edit a Helm Chart?")
-- The software *contained* in our Helm Charts (for example "how do I configure Radarr?")
-- Cluster Operations (maintenance, fixing flaky pods, tracking non-deploying fluxcd helm-releases/customisations)
-- How to install/use `kubectl`/`talosctl`
+:::
+
+## FluxCD Support Policy
+
+- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.
+- We offer [basic documentation](./guides/fluxcd) and support on how to create a `helm-release` file, to deploy our helm-charts on Flux.
+- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.
+
+## What's Explicitly Not Covered By Support
+
+- The functioning and compatibility of your self-designed operating systems and kubernetes distros, such as k3s.
+- Troubleshooting complete clusters, outside of just our Helm-Charts.
+- CloudFlare proxying.
+- The functioning and compatibility of your self-designed operating systems and kubernetes distros, such as k3s.
+- The software *contained* in our Helm Charts (for example "how to configure Radarr to connect to a torrent client?").
+- Trouble shooting complete clusters, outside of just our Helm-Charts
+- How to install additional tools (like: `kubectl`, `talosctl` etc)
```

Comment:
```suggestion
- How to install additional tools (like: `kubectl`, `talosctl` etc)
```

Maybe we can link kubectl and talosctl here explicitly? Might be a nice feature to add while at it anyway.